ReferenceDeployment/Top/ReferenceDeploymentTopology.cpp

```diff
@@ -171,20 +173,34 @@ void setupTopology(const TopologyState& state) {
 Os::Mutex cycleLock;
 volatile bool cycleFlag = true;
 
+void myIsr(U32 ticks) {
```

Comment:
You don't need this. The component can self-register `s_isr` from withing the public function I mentioned.